README.md

```diff
@@ -1,246 +1,223 @@
-2. **Select the Resource Group and Region**: Choose the resource group and region where the infrastructure was deployed. The resource group name is based on the prefix and resource_group_name inputs of the deployable architecture.
-3. **Select the Toolchain**: Select "RAG Sample App-CI-Toolchain"
-    ![toolchain](./images/min/8-toolchain.png)
-4. **Access the Delivery Pipeline**: In the toolchain view, select ci-pipeline under Delivery pipeline
-    ![toolchain](./images/min/9-pipeline.png)
-5. **View the CI Pipeline Status**: The current status of the CI pipeline execution can be found under the "rag-webhook-trigger" section.
 
-### Verifying the Application Deployment
+1.  Access the DevOps Toolchains view by navigating to [DevOps > Toolchains](https://cloud.ibm.com/devops/toolchains) in the target account.
+1.  Select the resource group and region where the infrastructure was deployed. The resource group name is based on the prefix and `resource_group_name` inputs of the deployable architecture.
+1.  Select the **RAG Sample App-CI-Toolchain**.
+    ![Toolchain](./images/min/8-toolchain.png)
+1.  In the toolchain view, select **ci-pipeline** in delivery pipeline.
+    ![Toolchain](./images/min/9-pipeline.png)
+5.  Look for the status of the CI pipeline execution in the **rag-webhook-trigger** section.
 
-Once the initial run of the CI pipeline complete, you should be able to view the application running in the created [Code Engine project](https://cloud.ibm.com/codeengine/projects).
-
-
-### Enabling Watson Assistant
+### Verifying the sample application
 
-After the application has been built and is running in Code Engine, there are additional steps specific to the sample app that need to be completed to fully enable Watson Assistant in the app. To complete the installation, follow the steps outlined in the [application README.md file](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-README.md).
+After the CI pipeline completes, you can access the application in the created [Code Engine project](https://cloud.ibm.com/codeengine/projects).
 
 
-## 6. Important Deployment Considerations
-
-### API Key Requirements
+### Enabling Watson Assistant
 
-The deployable architecture can only be deployed with an API Key associated with a user. It is not compatible with API Keys associated with a serviceId. Additionally, it cannot be deployed using the Project trusted profile support.
+After the application is built and is running in Code Engine, enable Watson Assistant in the app.
+To complete the installation, follow the steps that are outlined in the [Configuration steps for watsonx Assistant artifacts](https://github.com/IBM/gen-ai-rag-watsonx-sample-application/blob/main/artifacts/artifacts-README.md) file in the gen-ai-rag-watsonx-sample-application GitHub project.
```

Comment:
@brendankellyibm is this still necessary?

Reply:
I don't believe so - none of the Discovery pieces are applicable to Elastic Search installations, and we seem to have the assistant skills in the DA already. @ajainghub might be able to confirm. If so, we need to clean up that doc file too.